views/admin-page-settings.php

```diff
@@ -34,16 +34,10 @@
 							<?php /* phpcs:disable */ ?>
 							<?php /* echo esc_html( $tab['title'] ); */ ?>
 							<?php /* echo esc_html( $tab['desc'] ); */ ?>
+							<?php /* phpcs:enable */ ?>
```

Comment:
Cleanup
```suggestion
```

Reply:
:+1:

This was a WIP, for different tabs (sections) as we can add more via filter